### DISCUSSION TITLE
Color picker

### DISCUSSION THREAD

**Discussion by ghost** [FelixKratz/SketchyBar#181] [Q&A] [answered]
## Color picker   
`
Is there a available color picker for configuration files? I do not understand how the color codes in the configuration files work. Please help, thank you. `

**Comment by es183923** [comment, marked as answer]
No, there is not a color picker. You can use any normal color picker and it will work.

The color codes are very similar to standard color hex codes. The format is: 0x+opacity+red+green+blue, with red, green, blue, and opacity being a 2 character hex code.

Felix: Fixed Ordering

**Reply by ghost** [reply]
`Thank you`

**Reply by FelixKratz** [reply]
https://felixkratz.github.io/SketchyBar/config/types the color is an argb hex, so it is composed just as @es183923 described it. I will see if I can get a color picker working on the website.

**Comment by FelixKratz** [comment]
I have created a simple color picker for the website https://felixkratz.github.io/SketchyBar/config/tricks. Enjoy!
